src/lua/CommunityBalanceMod/Combined/Balance.lua

```diff
@@ -45,6 +45,21 @@ for _,chamberTraits in ipairs(kUpgradesGroupedByChamber) do
 end
 
 
+-- FortressPvE
+kFortressUpgradeCost = 24
+kFortressResearchTime = 10
+kFortressAbilityCooldown = 10
+kFortressAbilityCost = 3
+kCragCost = 8
+kShiftCost = 8
+kShadeCost = 8
+kShadeCost = 8
```

Comment:
kShadeCost is here twice, is that supposed to be kWhipCost? if not would probably be best to remove it

Reply:
Copy paste victim. Should have been Whip